src/babelhook.js

```diff
@@ -0,0 +1,41 @@
+const { babelSharedLoader } = require("../babel_config")
```

Comment:
I think almost all of this file is probably unnecessary for this project since it deals with testing infrastructure for react and DOM workarounds. Can we remove this file and the `--require babelhook` line in `js_test.sh`?